guides/typed-json/primitive-arrays-and-single-value-types/README.md

```diff
@@ -1,12 +1,34 @@
-# Primitive arrays and single value types
+# Primitive arrays and single value types 
```

Comment:
The current version of this page differs between `RC` and `Develop` branches. The things mentioned here is not implemented in Starcounter `RC` (`v2.3`). 

It looks like it's based on the page from Develop, [[RC]](https://docs.starcounter.io/v/RC/guides/typed-json/primitive-arrays-and-single-value-types/) [[Develop]](https://docs.starcounter.io/v/Develop/guides/typed-json/primitive-arrays-and-single-value-types/)